src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -203,13 +284,18 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         serverManagementApiUrl: {type: String, value: null},
         serverPortForNewAccessKeys: {type: Number, value: null},
         isAccessKeyPortEditable: {type: Boolean, value: false},
+        isAccessKeyDataLimitEnabled: {type: Boolean, notify: true},
+        accessKeyDataLimit: {type: Object, value: null}, // type: app.DisplayDataAmount
+        canSetAccessKeyDataLimit: {type: Boolean, value: false}, // Whether the server supports data limits.
```

Comment:
Rename to `supportsAccessKeyDataLimit", it's clearer.

Reply:
Done.